app/slackbot.go

```diff
@@ -34,6 +35,7 @@ func runDeployBot() {
 		publishedRegexp: publishes,
 	})
 
+	ctx := context.Background()
```

Comment:
This could be improved by passing in a context from `App.Start()` which has a deferred cancellation.
With this, the `select` statement can have a case for `<-ctx.Done()` to terminate more gracefully.